### PR TITLE
CM->RMC Retagging of IDs

### PR DESCRIPTION
**Pull request by FungiFellow**
## About the PR

I am going to change many ID's in these files from CM to RMC. I am not touching the code so much as a chunked rewrite of IDs. This is not an all encompassing rewrite but rather to reduce future workload

See https://github.com/RMC-14/RMC-14/pull/3583/files for details
## Why / Balance

Longstanding TODO's of changing stuff from CM to RMC. 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by deltanedas** [comment]
webedit ops oorah

**Comment by FungiFellow** [comment]
> webedit ops oorah

As long as I don't fuck up it's golden (I do webedit ops because I do not have the hardware to do this shit locally without it taking an hour for minor things)

File: Resources/migration.yml [hunk 1]
```diff
@@ -711,3 +711,5 @@ CMCrateSecurePhoron: RMCCrateSecurePhoron
 CMBoxBodyBag: RMCBoxBodyBag
 CMBoxLatexGloves: RMCBoxLatexGloves
 CMBoxSyringe: RMCBoxSyringe
+CMBoxSterileMask: RMCBoxSterileMask
+CMBoxPillCanister: RMCBoxPillCanister
```
Comment (QuietlyWhisper) [inline Resources/migration.yml:715]: dupe of line 682

Comment (FungiFellow) [reply Resources/migration.yml:722]: Thanks

**Review by DrSmugleaf — APPROVED** [review]
holy xhit

**Pull request merged**